src/checkpoint/checkpoint.h

```diff
@@ -197,15 +197,16 @@ extern int __wt_checkpoint_server_destroy(WT_SESSION_IMPL *session)
   WT_GCC_FUNC_DECL_ATTRIBUTE((warn_unused_result));
 extern int __wt_checkpoint_sync(WT_SESSION_IMPL *session, const char *cfg[])
   WT_GCC_FUNC_DECL_ATTRIBUTE((warn_unused_result));
+extern void __wt_checkpoint_apply_or_skip_handle_update(WT_SESSION_IMPL *session, uint64_t time_us);
```

Comment:
```suggestion
extern void __wt_checkpoint_stats_handle_update(WT_SESSION_IMPL *session, uint64_t time_us);
```
Just my suggestion, feel free to reject.